x-pack/plugins/lists/server/config.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { TypeOf, schema } from '@kbn/config-schema';
+
+export const ConfigSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: true }),
+  listIndex: schema.string({ defaultValue: '.lists' }),
+  listItemIndex: schema.string({ defaultValue: '.items' }),
```

Comment:
Should we prefix with list to make this less ambiguous? e.g. `.list-items`? Which begs to ask, should we scope it even further to `.security-lists` and `.security-list-items`?

I only ask as when I was putting on my different testing 🎩's I pondered what other developers might think when coming across a system index named `.items`... 🤔

Reply:
I was careful to not add a prefix that is the same because sometimes people do bad things where they would do: `.security-*` or `.list-*` and then they would end up mixing both indexes together which is what I wouldn't want to trouble shoot. I kind of like them disjointed even though it reduces discoverability.

I am still open though to changing it if maybe I am thinking about this wrong.

Reply:
Ahh yes, that makes sense to me. I think the counter to that is you're kind of on your own when creating aliases to system indices, but I don't feel strongly one way or the other so am fine keeping as is.